tools/frr.in

```diff
@@ -533,9 +524,8 @@ convert_daemon_prios
 
 if [ ! -d $V_PATH ]; then
 	echo "Creating $V_PATH"
-	mkdir -p $V_PATH
-	chownfrr $V_PATH
-	chmod 755 /$V_PATH
+	install -g "$FRR_GROUP" -o "$FRR_USER" -m "$FRR_CONFIG_MODE" -d /proc "$V_PATH"
```

Comment:
Why is there a `/proc` here? There's no "source" directory with the `-d` option, this is trying to create `/proc` now

Reply:
Gah, leftovers... Will remove.